spell/spell-reviewer-goerli-checklist.md

```diff
@@ -70,6 +70,20 @@ Spell Actions (Per Exec Sheet):
       * [ ] Exceptions must follow archive patterns (e.g. `GemLike` for `MKR`)
     * [ ] Visibility is `internal`
     * [ ] State mutability is `immutable`
+* [ ] Ensure the comments inside the spell action are correct:
```

Comment:
I would actually move it up, to be right before `* [ ] Local Environment Actions`